config/log.conf

```diff
@@ -19,5 +19,5 @@ log_format main_json escape=json
     '"time_local":"$time_local"'
   '}';
 
-access_log /dev/stdout main_default;
+access_log /dev/stdout main_json;
```

Comment:
I dunno if changing the default logging is agreeable but figured in 2021 we might as well move to structured logging and so having this in JSON is a good default! 😅 Makes parsing it easier!!

Reply:
And I should have mentioned that the reason I left `main_default` in was because it allows one to override `access_log /dev/stdout main_default;` elsewhere e.g. like in a `location` block.

Reply:
>  dunno if changing the default logging is agreeable but figured in 2021 we might as well move to structured logging and so having this in JSON is a good default! 😅 Makes parsing it easier!!

It's bittersweet after seeing human-readable lines for so long. 😭

Reply:
> It's bittersweet after seeing human-readable lines for so long. 😭

🙈 🙊 🙉